app/assets/stylesheets/layouts/footer.scss

```diff
@@ -0,0 +1,31 @@
+footer {
+  position: relative;
+
+  .row {
+    color: $element-toned-down;
+    font-size: 18px;
+    height: 96px;
+
+    @include media-breakpoint-down(sm) {
+      margin-bottom: 0;
+      .copyright-section {
+        font-size: 12px;
+      }
+    }
+  }
+
+  .branding-section{
+    letter-spacing: 2px;
+    font-size: 36px;
+    font-weight: 900;
+  }
+}
+
+.login {
+  footer  {
+   .branding-section {
+     color: $text-light;
```

Comment:
This feels like weird nesting...why in login? And why different nesting from above?

Reply:
its white on login page and gray on everywhere else

Reply:
@gcwokeup If that's the case we'd use BEM to do apply the M part (modifier). So you'd do something like `.branding-section--white` to override the default state and set it white. Don't want to set that based on hierarchy because if the html structure changes or a higher order class name changes then you'd break it.